src/directives/vue.js

```diff
@@ -69,7 +71,7 @@ function register(ngModule) {
 
         _.forEach(syncedPropertiesMapping, (ngProp, vueProp) => { // .sync
           vm.$children.forEach((c) => c.$on(`update:${vueProp}`, (value) => {
-            console.log(`vue(${vueProp}) => ng(${ngProp}) =`, value);
+            if(options.verbose) log(`vue(${vueProp}) => ng(${ngProp}) =`, value);
```

Comment:
You dropped `console` by mistake in from of `log`

Reply:
BTW thank you for doing that i wanted to do it long time ago

Reply:
done